src/GitHubExtension/DataModel/GitHubDataStoreSchema.cs

```diff
@@ -13,7 +13,7 @@ public GitHubDataStoreSchema()
     }
 
     // Update this anytime incompatible changes happen with a released version.

```

Comment:
Do we have a definition of "Incompatible changes" somewhere? I imagine it's anytime the "schema" changes, but what counts as that?